src/main/java/spaceinvaders/spaceinvaders_framework/Game.java

```diff
@@ -76,10 +79,10 @@
     /**
      * Vector to store all alien, bullet and barrier objects
      */
-    private Vector<Alien> aliens;
-    private Vector<Bullet> alienBullets;
-    private Vector<Bullet> shipBullets;
-    private Vector<Barrier> barriers;
+    public Vector<Alien> aliens;
+    public Vector<Bullet> alienBullets;
+    public Vector<Bullet> shipBullets;
+    public Vector<Barrier> barriers;
```

Comment:
It's not a good idea to make all these vectors public. That makes the game class head towards a god class again and violates encapsulation rules.